packages/stripe_platform_interface/lib/src/method_channel_stripe.dart

```diff
@@ -427,6 +441,28 @@ class MethodChannelStripe extends StripePlatform {
     return isSupported ?? false;
   }
 
+  @override
+  Future<AvailableMobilePayOptions> availableMobilePayOptions({
```

Comment:
what is the use of this method? we have a function called: `isPlatformPaySupported`